main.py

```diff
@@ -2,9 +2,47 @@
 
 from db.models import Race, Skill, Player, Guild
 
+import json
+
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        data = json.load(file)
+
+    for nickname, info in data.items():
+        if not Race.objects.filter(name=info["race"]["name"]).exists():
+            race = Race.objects.create(
+                name=info["race"]["name"],
+                description=info["race"]["description"]
+            )
+        race = Race.objects.get(name=info["race"]["name"])
```

Comment:
can you optimize it? At least, you can make `race = Race.objects.create()...`. 
Then, you don't have to make an additional request to the database to get that value
Now the difference is not felt much, but in real projects, it is VERY important. 
Btw, It is best practice to use the `get _or _create ()` method in general, as this, will improve readability
That is, you MUST reduce the number of requests to the database, but not necessarily use `get_or_create()`(that is only best practice for now)